src/main/java/com/java_essential/kopaiev/essential/homework/lesson2/task2/Car.java

```diff
@@ -0,0 +1,26 @@
+package com.java_essential.kopaiev.essential.homework.lesson2.task2;
+
+public class Car {
+    private int year;
+    private String color;
+
+    public Car()
+    {
```

Comment:
Please, don't do that)

public Car() {}

Reply:
fixed